brainscore/benchmarks/README.md

```diff
@@ -0,0 +1,7 @@
+# Brain-Score Benchmarks
+
+**File structure**: One file per assembly with all the benchmarks inside.
+                    This file contains the loading of the assembly (with eventual processing) and the benchmarks.
+
+**Versioning**: Each benchmark is numbered by a version number. 
```

Comment:
For submissions, I think we should just always use the latest *release*.
We still track everything in git versions (including minor changes that don't affect scores), but for major changes that will change the scores, I think we should explicitly bump the versions so that users can compare their scores based on the version number.